root/etc/e-smith/templates/etc/ssh/sshd_config/70Restricted2Sftp

```diff
@@ -0,0 +1,23 @@
+{
+    # restrict groups to sftp, ssh access is refused
+    my $domain = $DomainName || die ('Cannot retrieve DomainName');
```

Comment:
Please remove the `die`. If `DomainName` is not set, also `sshd` is not set ;)
```suggestion
    my $domain = $DomainName;
```

Reply:
ok